test/resources/load_data/jsonline.jl

```diff
@@ -0,0 +1,2 @@
+{"col1":true,"col2":1,"col3":"var","col4":"2020-09-07","col5":"2020-09-07 00:00:00","col6":"2020-09-07 00:00:00","col7":"18","col8":121.11}
+{"col1":"true","col2":"1","col3":"var","col4":"2020-09-07","col5":"2020-09-07 00:00:00","col6":"2020-09-07 00:00:00","col7":"18","col8":"121.11"}
```

Comment:
Test a case that {"col1": 1, "col2": "OK"}     {"col2": "OK", "col1": 1}

Reply:
OK